File: src/test/scala/cern/colt/matrix/tfloat/impl/SparseRCFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.SparseRCFloatMatrix2D

class SparseRCFloatMatrix2DViewTest(arg0: String) extends SparseRCFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseRCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseRCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseRCFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}

File: src/test/java/cernx/colt/matrix/tfloat/impl/SparseRCFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class SparseRCFloatMatrix2DViewTest extends SparseRCFloatMatrix2DTest {

    public SparseRCFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
